packages/core-entities/src/client/accounts/current-account.ts

```diff
@@ -26,6 +26,13 @@ import { AccountDefinition, AccountIncentiveDebt } from '../../Definitions';
 import { fetchUsingMulticall } from '../../server/server-registry';
 import { SNOTEWeightedPool } from '../../exchanges';
 
+interface StakeNoteStatus {
```

Comment:
This should move to the `Definitions.ts` file and replace the type in `AccountDefinition`